### ISSUE TITLE
Command

### ISSUE THREAD

**Issue by emetik** [A5hleyRich/wp-cli-unsplash-command#1]
the correct call would be  : 
`wp unsplash generate --media_date=random --count=100 --media_dimensions=1600x1200
instead of 
`wp media generate --media_date=random --count=100 --media_dimensions=1600x1200 `

**Comment by A5hleyRich** [comment]
Good catch. I changed the command last minute, but the docs don't reflect that. I'll update.
